packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html

```diff
@@ -0,0 +1,447 @@
+                            ></button>
+                        </dt>
+                        <oui-tile-description>
+                            {{::
+                            'pci_projects_project_training_dashboard_cli_integration_description'
+                            | translate }}
+                        </oui-tile-description>
+                    </dl>
+                    <oui-tile-actions>
+                        <oui-dropdown placement="end">
+                            <oui-dropdown-trigger
+                                text="{{ ::'pci_projects_project_training_dashboard_cli_integration_details' | translate }}"
+                            ></oui-dropdown-trigger>
+                            <oui-dropdown-content>
+                                <a
+                                    href="{{ $ctrl.currentRegion.documentationUrl }}"
+                                    target="_blank"
+                                    rel="noopener"
+                                    class="oui-dropdown-option ng-scope"
+                                >
+                                    {{
+                                    ::'pci_projects_project_training_dashboard_cli_integration_doc'
+                                    | translate }}
+                                    <span
+                                        class="oui-icon oui-icon-external_link"
+                                        aria-hidden="true"
+                                    ></span>
+                                </a>
+                                <oui-dropdown-item
+                                    href="{{ :: $ctrl.installLink }}"
+                                    >{{
+                                    ::'pci_projects_project_training_dashboard_cli_integration_install'
+                                    | translate }}</oui-dropdown-item
+                                >
+                            </oui-dropdown-content>
+                        </oui-dropdown>
+                    </oui-tile-actions>
+                </div>
+            </oui-tile>
+        </div>
+        <div class="col-md-3">
+            <oui-tile data-heading="Usage" class="h-100">
+                <oui-tile-definition>
+                    <oui-tile-description>
+                        <div class="row d-lg-flex">
+                            <div class="col-md-6">
+                                <span class="oui-status oui-status_info">
+                                    <span
+                                        >{{ ::
+                                        'pci_projects_project_training_jobs_state_RUNNING'
+                                        | translate }}</span
+                                    >
+                                </span>
+                                <h2>{{ :: $ctrl.nbRunning }}</h2>
```

Comment:
i don't think h2 is suitable here, maybe just use some css class if you want larger text

Reply:
fixed